newspaper/text.py

```diff
@@ -156,3 +156,28 @@ def get_stopword_count(self, content):
         ws.set_stopword_count(len(overlapping_stopwords))
         ws.set_stop_words(overlapping_stopwords)
         return ws
+
+
+class StopWordsHindi(StopWords):
```

Comment:
I just realized that `StopWordsKorean` and your new `StopWordsHindi` dont' need separate classes, both classes reuse 100% of the code from `StopWords` - it's fine I will merge this PR but assign myself a task to dedupe the code later